lib/ansible.js

```diff
@@ -24,11 +25,20 @@ AbstractAnsibleCommand.prototype.exec = function(options) {
 
   var deferred = Q.defer();
   var processOptions = {};
-  if (options && options.cwd) {
+  if (!options) options = {};
+
+  if (options.cwd) {
     processOptions.cwd = options.cwd;
   }
 
+  // Make Ansible output unbuffered so the stdout and stderr `data` events 
+  // pickup output up more regularly
+  var unbuffered = (options.buffered) ? '' : 1;
+  processOptions.env = process.env;
+  _.extend( processOptions.env, { PYTHONUNBUFFERED: unbuffered } );
+
   var output = '';
+  debug('exec',this.commandName(), this.compileParams(), processOptions);
```

Comment:
as a side node, the lack of verbosity, or control of it, is the main feature I like about the debug library. The `debug` call is a noop unless you specifically ask for that debug string/class to be turned on so it works really well for multi file/class apps, when you setup the naming structure.  It's always a nice surprise when you turn DEBUG='*' on and your dependencies start doing debug as well.